performance-test/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "2"
```

Comment:
This docker have the real database servers (Cassandra and MySQL) to be used in tests.